src/extension/types/typedefs.js

```diff
@@ -33,9 +33,10 @@
 
 /**
  * @typedef {Object} OptionsConfig
+ * @prop {string} id The id of the config
  * @prop {string} [project] The name of the project
  * @prop {string} giturl The url to the repository
- * @prop {string} mountpoint The content source URL
+ * @prop {string[]} mountpoints The content source URL
```

Comment:
Change things to send in the mountpoint array into app (instead of pulling out the first mountpoint in content.js). This is just updating the type to go along with that change.